soh/src/code/z_message_PAL.c

```diff
@@ -951,7 +955,10 @@ void Message_DrawText(GlobalContext* globalCtx, Gfx** gfxP) {
                         }
                     }
                     i = j - 1;
-                    msgCtx->textDrawPos = i + 1;
+                    if (CVar_GetS32("gFastText", 0) !=0)
+                        msgCtx->textDrawPos = i + 5;
```

Comment:
any chance the `5` here could be changed to a const or something to avoid magic numbers?

or maybe even switching away from using a bool to having a slider for different text speeds

Reply:
Yep, totally could, I think I've fixed that now?

I'll take a look at making it a slider, it should be possible.